mxcubeweb/routes/signals.py

```diff
@@ -654,6 +655,7 @@ def beam_changed(*args, **kwargs):
             "size_x": _beam[0],
             "size_y": _beam[1],
             "shape": _beam[2].value,
+            "aperture": _beam[3],
```

Comment:
Ok, so to avoid future conflicts it would probably be better if this is:

```
            "shape": _beam[2].value,
            "label": _beam[3],
        }
```